packages/footer/src/sass/module.scss

```diff
@@ -0,0 +1,74 @@
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// footer module
+// Footers help users who reach the bottom of a page without finding what they want.
+//
+// Content:
+// - Footer styles
+//   -Basic footer block styling
+//   - Footer `nav` navigation styling (supports up to 4 link blocks)
+//   - Footer bottom section styling
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+
+@import 'globals.scss';
+
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// Footer styles
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+/**
+ * Basic footer block styling.
+ */
+.uikit-footer {
+	background-color: $uikit-colour-Background-20;
+	border-top: 6px solid $uikit-colour-Beta;
+	padding: uikit-space( 1 ) 0;
+}
+
+
+/**
+ * Footer `nav` navigation styling.
+ */
+.uikit-footer__navigation {
+	margin: 0 uikit-space( 1 ) uikit-space( 1 );
+	padding-bottom: uikit-space( 1 );
+	border-bottom: $uikit-border;
+
+	.uikit-footer__navigation__list {
+		padding-left: 0; //overrides: links-lists
+		list-style: none;
+
+		.uikit-footer__navigation__list-group {
```

Comment:
this doesnt need to be nested 3 deep it could be `.uikit-footer__navigation-list-group`